beacon-chain/rpc/eth/validator/handlers_block.go

```diff
@@ -182,13 +190,28 @@ func (s *Server) ProduceBlockV3(w http.ResponseWriter, r *http.Request) {
 	}
 
 	s.produceBlockV3(ctx, w, r, &eth.BlockRequest{
-		Slot:         primitives.Slot(slot),
-		RandaoReveal: randaoReveal,
-		Graffiti:     graffiti,
-		SkipMevBoost: false,
+		Slot:               primitives.Slot(slot),
+		RandaoReveal:       randaoReveal,
+		Graffiti:           graffiti,
+		SkipMevBoost:       false,
+		BuilderBoostFactor: bbFactor,
 	}, any)
 }
 
+func processBuilderBoostFactor(raw string) (*wrapperspb.UInt64Value, error) {
```

Comment:
The spec defines the boost factor as `Uint64`. There is no need for this function, you can just call `shared.UintFromQuery`

Reply:
I need it as nullable, i'll doublecheck sharedUintFromQuery though